packages/webview_flutter/webview_flutter_android/example/android/settings.gradle

```diff
@@ -20,7 +20,7 @@ pluginManagement {
 plugins {
     id "dev.flutter.flutter-plugin-loader" version "1.0.0"
     id "com.android.application" version "8.5.1" apply false
-    id "org.jetbrains.kotlin.android" version "1.7.10" apply false
+    id "org.jetbrains.kotlin.android" version "1.9.10" apply false
```

Comment:
Judging from roller failures ([example](https://github.com/flutter/packages/pull/8147/checks?check_run_id=33349404085)) I think this needs to be done in `webview_flutter`'s example as well now that this has been published.

Reply:
Thanks for fixing this!